e2e-tests/docs/CI.md

```diff
@@ -0,0 +1,135 @@
+# CI Steps and Processes for Testing
+
+This document provides a comprehensive overview of our Continuous Integration (CI) testing processes. It explains when and how different test jobs are triggered, where to monitor them, and details about the environments hosting these tests, along with guidance on how to access them.
+
+## GitHub Pull Requests
+
+When a new Pull Request (PR) is opened at [backstage-showcase](https://github.com/janus-idp/backstage-showcase), tests can be triggered in two ways:
+
+1. **Commenting `/ok-to-test`**: Only members of the janus-idp GitHub organization can mark the PR with this comment to validate it for testing.
```

Comment:
e2e-tests runs as soon as pr is opened and `/ok-to-test` is required only if the PR is raised by someone who isn't a member of janus-idp org.

Reply:
Just updated it, thank you @subhashkhileri for your comments!